spec/models/variant_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'spec_helper'
+
+module Spree
+  RSpec.describe Variant do
+    let(:product) do
+      Product.create!(
+        name: 'globalize',
+        price: 19.99,
+        shipping_category: ShippingCategory.create!(name: 'a')
+      )
+    end
+
+    let!(:variant) do
+      Variant.create!(
+        price: 19.99,
+        product: product
+      )
+    end
+
+    it 'fetches variant from product via translation table' do
+      variants = described_class.includes(:product).ransack(name_cont: 'globalize').result.to_a
+      expect(variants.last).to eq variant
```

Comment:
I wonder if we need the same code as in https://github.com/spree-contrib/spree_i18n/pull/595/files#diff-5d3646eb5b08660b00d9f57891773b27R20.

I'd expect the build to fail though since this will only be fixed on next rails release but it feels relevant to leave the spec here (could point back to 4-2-stable) so we know whether variants search would work or not. Can you tell if the variants search on admin/orders work on latest rails gem release and with this patch?